modules/ROOT/pages/license-influx-update.adoc

```diff
@@ -0,0 +1,47 @@
+= Configure Backup and Restore for Anypoint Private Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== To Update the InfluxDB License
```

Comment:
This section was redone in a simpler way in the runbook: https://pce-runbook.kbuild.msap.io/content/amv-addon-3.0.x/influxdb/influx-license-update.html

I think it makes sense to move it here since it's something that the customer could want to do

Reply:
Agree - Deleted